pages/common/git-checkout.md

```diff
@@ -4,12 +4,16 @@
 
 - Switch to another branch:
 
-`git checkout {{BRANCH-NAME}}`
+`git checkout {{branch_name}}`
 
 - Create and switch to a new branch:
 
-`git checkout -b {{BRANCH-NAME}}`
+`git checkout -b {{branch_name}}`
 
 - Undo unstaged local modification:
 
 `git checkout .`
+
+- Checkout a file from another branch:
```

Comment:
@agnivade I agree with your concern but that wording can be a little confusing IMO. How about: "Replace a file in the current working directory with the version of it committed in a given branch"?